lib/types/entry.types.ts

```diff
@@ -12,11 +12,17 @@ export interface TaskInputData {
   dueDate?: string
 }
 
+export interface BasicQueryOptions {
```

Comment:
Since we also allow `order`, we can also just use `SearchQuery` from [here](https://github.com/contentful/ui-extensions-sdk/blob/12cbe1a8f27e56dfb3a0764ee89c3377be3ab6f5/lib/types/utils.ts#L64), right?